react/features/base/lib-jitsi-meet/actions.web.js

```diff
@@ -0,0 +1,61 @@
+import {
+    LIB_DISPOSED,
+    SET_CONFIG
+} from './actionTypes';
+import './middleware';
+import './reducer';
+
+/**
+ * Disposes lib-jitsi-meet.
+ *
+ * @returns {Function}
+ */
+export function disposeLib() {
+    // XXX We're wrapping it with Promise, because:
+    // a) to be better aligned with initLib() method, which is async.
+    // b) as currently there is no implementation for it in lib-jitsi-meet, and
+    // there is a big chance it will be async.
+    // TODO Currently, lib-jitsi-meet doesn't have any functionality to
+    // dispose itself.
+    return dispatch => {
+        dispatch({ type: LIB_DISPOSED });
+
+        return Promise.resolve();
+    };
+}
+
+/**
+ * Initializes lib-jitsi-meet with passed configuration.
+ *
+ * @returns {Function}
+ */
+export function initLib() {
+    return (dispatch, getState) => {
+        const config = getState()['features/base/lib-jitsi-meet'].config;
+
+        if (!config) {
+            throw new Error('Cannot initialize lib-jitsi-meet without config');
+        }
+
+        // XXX Temporary solution. Until conference.js hasn't been moved
+        // to the react app we shouldn't use JitsiMeetJS from react app.
+        return Promise.resolve();
```

Comment:
The file react/features/base/lib-jitsi-meet/actions.web.js is practically a copy of ~55 lines from  react/features/base/lib-jitsi-meet/actions.native.js with a modification of ~10 lines. As per the coding style, this is unacceptable.